src/sql/workbench/browser/modelComponents/media/toolbarLayout.css

```diff
@@ -50,8 +50,8 @@
 }
 
 .modelview-toolbar-container .modelview-toolbar-component modelview-button .monaco-text-button.icon {
-	padding-left: 15px;
-	background-size: 11px;
+	padding-left: 19px;
```

Comment:
Does this align with what we have in the Notebook toolbar? Long term it'd be great if these could share identical CSS at least.

Reply:
Unfortunately it looks like no - that uses the carbon taskbar styles https://github.com/microsoft/azuredatastudio/blob/db498db0a82564fb4e813b794b16c73d5d8db0eb/src/sql/base/browser/ui/taskbar/media/taskbar.css#L47 so there isn't really anything shared there at the moment. Would be nice to align on this though in the future, I agree.